pkgs/development/web/deno/default.nix

```diff
@@ -58,8 +58,8 @@ rustPlatform.buildRustPackage rec {
 
     installShellCompletion --cmd deno \
       --bash <($out/bin/deno completions bash) \
-      --fish <($out/bin/deno completions fish) \
-      --zsh <($out/bin/deno completions zsh)
+      --zsh <($out/bin/deno completions zsh) \
+      --fish <($out/bin/deno completions fish)
```

Comment:
I guess this part can be reverted, right?

Reply:
Ah, I had to remove it since `1.5.3` until fixed in `1.6.1` but it I put it back in the wrong way around. I'll amend that